src/components/form/trash-list.vue

```diff
@@ -63,6 +73,7 @@ export default {
   },
   created() {
     this.fetchDeletedForms(false);
+    this.fetchUserPreferences();
```

Comment:
I think we should focus on user-level settings for now and come back to session-specific settings if we see the need!

I think it will be unusual for the same user to pick different values for the same setting in different sessions. I think it's relatively likely that a user would change different settings in different sessions. That second situation feels like it would be handled well by a `PATCH` endpoint to only update one setting at a time. I haven't looked at any code yet so that may already be happening! I think writing all possible user settings at once to the server would make that second case too confusing.

Reply:
This behavior sounds great to me! I agree there could be some cases where the merging could be surprising in a negative way but I think pleased/satisfied/didn't notice anything because it behaved as expected will be most common reactions.

Yes, let's document this. How about filing an issue at https://github.com/getodk/docs/issues for all of this related work and keeping notes or a checklist of interesting things to include? These can be links to comments or pieces of code, whatever you think will best jog our memory once we write the docs. Thanks!